tests/test_hepnames.py

```diff
@@ -1437,7 +1437,10 @@ def test_stub_from_980__a_useful():
     ]
     result = hepnames2marc.do(result)
 
-    assert expected == result['980']
+    for el in expected:
+        assert el in result['980']
+    for el in result['980']:
+        assert el in expected
```

Comment:
Now that we're no longer appending in a post processing filter the ordering is no longer guaranteed. Checking for double inclusion works in Python 3.6 too, unlike asserting the equality on the two `sorted` lists.